util/stmtsummary/statement_summary.go

```diff
@@ -257,6 +263,9 @@ func (ssMap *stmtSummaryByDigestMap) AddStatement(sei *StmtExecInfo) {
 		if !ssMap.Enabled() {
 			return nil, 0
 		}
+		if sei.IsInternal && !ssMap.EnabledInternal() {
```

Comment:
Also judge it in `adaptor.go`. It's double check here, just to avoid lock problems.

Reply:
done